tests/planner/vectorfield/test_VectorFieldPlanner.cpp

```diff
@@ -1,16 +1,19 @@
-﻿#include <gtest/gtest.h>
 #include <tuple>
+
 #include <dart/dart.hpp>
-#include "aikido/planner/dart/ConfigurationToEndEffectorOffset.hpp"
+#include <gtest/gtest.h>
+
 #include <aikido/constraint/Testable.hpp>
 #include <aikido/distance/defaults.hpp>
+#include <aikido/planner/dart/ConfigurationToEndEffectorOffset.hpp>
 #include <aikido/planner/vectorfield/MoveEndEffectorOffsetVectorField.hpp>
 #include <aikido/planner/vectorfield/VectorFieldConfigurationToEndEffectorOffsetPlanner.hpp>
 #include <aikido/planner/vectorfield/VectorFieldPlanner.hpp>
 #include <aikido/planner/vectorfield/VectorFieldUtil.hpp>
 #include <aikido/statespace/GeodesicInterpolator.hpp>
 #include <aikido/statespace/SO2.hpp>
 #include <aikido/statespace/dart/MetaSkeletonStateSpace.hpp>
+
 #include "../../constraint/MockConstraints.hpp"
```

Comment:
@madan96: Seems like relative paths have not been fixed in tests. Is that possible to address?